pages/docs/tooling/tooling-aste.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Artificial Solver Testing Environment (ASTE)
+permalink: tooling-aste.html
+keywords: tooling, aste
+---
+
+## Motivation
+
+ASTE is a tool which can be used to understand the dynamics of preCICE, how the data is goes in and out. It consist of C++ and Python modules.
+
+## Installation
+
+Please make sure that dependencies are installed.
+
+### C++ modules
+
+#### Required
+
+- preCICE
+- MPI
+- CMake
+
+##### Optional
+
+- METIS
+
+### Python modules
+
+#### Required
+
+- NumPy
+- vtk (Visualization Toolkit)
```

Comment:
We should add some 'how to install' instructions here as well. Usually, python dependencies are given in a requirements file in the respective repository.